<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shownote</class>
 <widget class="QWidget" name="shownote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Deine Notizen&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>561</width>
     <height>401</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>559</width>
      <height>399</height>
     </rect>
    </property>
    <widget class="QListWidget" name="listview_shownote">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>401</height>
      </rect>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="openButton_shownote">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>130</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Öffnen</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::WindowNew"/>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton_shownote">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Zurück ins Hauptmenü</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoHome"/>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton_shownote">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Suchen</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditFind"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
